syntax = "proto3";

package protocol;

message Action {
    string action = 1;
}

message Data {
    float speed = 1;
    float distance = 2;
    int32 ammo = 3;
    float gun_angle = 4;
    int32 protect = 5;
    float machine_turn = 6;
    float gun_turn = 7;

    message coords {
        float x = 1;
        float y = 2;
    }

}
